FirebaseAuthUI/FIRPasswordSignInViewController.h

```diff
@@ -24,31 +24,59 @@ NS_ASSUME_NONNULL_BEGIN
     @brief The view controller that asks for user's password.
  */
 @interface FIRPasswordSignInViewController : FIRAuthUIBaseViewController
+{
+  @protected
+  /** @var _email
+   @brief The @c email address of the user from the previous screen.
+   */
+  NSString *_email;
```

Comment:
I don't want to expose this property to any other classes; only parent classes should have access to it.